CesiumGltf/include/CesiumGltf/MetadataConversions.h

```diff
@@ -363,15 +460,8 @@ template <> struct MetadataConversions<float, std::string_view> {
     // (which _is_ guaranteed to be null terminated) before parsing.
     // * except std::from_chars, but compiler/library support for the
     //   floating-point version of that method is spotty at best.
-    std::string temp(from);
-
-    char* pLastUsed;
-    float parsedValue = std::strtof(temp.c_str(), &pLastUsed);
-    if (pLastUsed == temp.c_str() + temp.size() && !std::isinf(parsedValue)) {
-      // Successfully parsed the entire string as a float.
-      return parsedValue;
-    }
-    return std::nullopt;
+    return MetadataConversions<float, std::string>::convert(
```

Comment:
The comment block preceding this line needs to be moved to line 430.

Reply:
This comment block initially annotated the conversion from `std::string_view` -> `float`. The point was that, due to a strange bug, you could not convert from the `std::string_view` directly and had to copy the data to a `std::string`. This is why I kept it in the `std::string_view` function, because the `std::string_view` must still be converted to a `std::string` before being converted to a `float`.